examples/extract_data/src/extract_data_login.py

```diff
@@ -0,0 +1,72 @@
+import os
+from airtop import AsyncAirtop, SessionConfigV1
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# Setup environment variables
+api_key = os.getenv("AIRTOP_API_KEY")
+profile_id = os.getenv("profile_id")
```

Comment:
For consistency we should either rename it to PROFILE_ID in caps and add it to the .env.example
```suggestion
profile_id = os.getenv("PROFILE_ID")
```
or allow the user to input it in the CLI.